could you help to check the build log @formatCvt 

at https://travis-ci.org/cocos2d/cocos2d-x/jobs/479738001
```
:cocos_new_test:compileDebugJavaWithJavac<==-----------> 20% EXECUTING [7s]> :cocos_new_test:compileDebugJavaWithJavac<==-----------> 20% EXECUTING [8s]> :cocos_new_test:compileDebugJavaWithJavac> Task :cocos_new_test:compileDebugJavaWithJavac FAILED
/home/travis/build/cocos2d/cocos2d-x/cocos_new_test/proj.android/app/src/org/cocos2dx/cpp/AppActivity.java:47: error: package Build does not exist
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
                 ^
/home/travis/build/cocos2d/cocos2d-x/cocos_new_test/proj.android/app/src/org/cocos2dx/cpp/AppActivity.java:47: error: package Build does not exist
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
                                          ^
2 errors
```

--- a/templates/cpp-template-default/proj.android/app/src/org/cocos2dx/cpp/AppActivity.java
+++ b/templates/cpp-template-default/proj.android/app/src/org/cocos2dx/cpp/AppActivity.java
@@ -26,6 +26,7 @@ package org.cocos2dx.cpp;
 
 import android.os.Bundle;
 import org.cocos2dx.lib.Cocos2dxActivity;
+import android.os.Build;
 import android.view.WindowManager;
 import android.view.WindowManager.LayoutParams;
 

